### PR TITLE
feat: Issue #91 add user posts in profile page

### PR DESCRIPTION
**Pull request by iSubhamMani**
## Related Issue

Issue #91 

## Description

Show user posts in the profile page if the user has uploaded a post

## Screenshots

Screenshot from 2024-01-02 10-18-25
Screenshot from 2024-01-02 10-18-34

### CODE REVIEW

File: src/components/pages/user/index.tsx
Comment (Sanchitbajaj02) [inline src/components/pages/user/index.tsx:1]: Can you display other data like fullname and username on this page along with email?

File: src/components/pages/user/userPosts/index.tsx [hunk 1]
```diff
@@ -1,28 +1,32 @@
 import { Heart, MessageCircle } from "react-feather";
+import { useSelector } from "react-redux";
+
+export default function UserPosts({ userId }: { userId: any }) {
+
+  const userPosts = useSelector((store : any) => store.posts.posts).filter((post:any) => post.accountId === userId);
```
Comment (Sanchitbajaj02) [inline src/components/pages/user/userPosts/index.tsx:6]: Please add type for `filter((post:any)`

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by iSubhamMani** [comment]
Screenshot from 2024-01-02 17-55-43

Is it okay ?

**Comment by Sanchitbajaj02** [comment]
> Screenshot from 2024-01-02 17-55-43
> 
> Is it okay ?

Yes it is for now. Will plan more issues with proper ui design soon

**Review by Sanchitbajaj02 — APPROVED** [review]

**Pull request merged**